src/nbl/CMakeLists.txt

```diff
@@ -711,6 +712,9 @@ if(NOT NBL_STATIC_BUILD)
 	add_dependencies(Nabla generate_nabla_defines)
 endif()
 
+add_dependencies(Nabla DeviceHeaders)
```

Comment:
move those 2 at the end of [`tools/CMakeLists.txt`](https://github.com/Devsh-Graphics-Programming/Nabla/pull/643/files#diff-70523fd3fe726ff8a7cba389b1deb5a5b761e02ebefe2fad6a1531d529e440dc) file

Reply:
Done